config/deploy.rb

```diff
@@ -9,18 +9,35 @@
 task :shared_db do
   on roles(:all) do
     execute "mkdir -p #{fetch(:deploy_to)}/shared/db/ && touch #{fetch(:deploy_to)}/shared/db/development.sqlite3"
+    execute "mkdir -p #{fetch(:deploy_to)}/static"
     execute "cp #{fetch(:deploy_to)}/static/.env.development.local #{fetch(:release_path)}/ || true"
   end
 end
 
 task :start_local do
   on roles(:all) do
     execute "export PATH=$PATH:/usr/local/bin && cd #{fetch(:deploy_to)}/current/script && source start_local.sh"
+    execute "mkdir -p #{fetch(:deploy_to)}/static"
   end
 end
 
 task :start_curly do
   on roles(:all) do
     execute "export PATH=$PATH:/usr/sbin/ && cd #{fetch(:deploy_to)}/current/script && chmod u+x * && source start_curly.sh"
+    execute "mkdir -p #{fetch(:deploy_to)}/static"
+  end
+end
+
+task :init_qp do
+  on roles(:all) do
+    execute 'gem install --user-install bundler'
+    execute "mkdir -p #{fetch(:deploy_to)}/static"
+    execute "cp #{fetch(:deploy_to)}/static/.env.production.local #{fetch(:release_path)}/ || true"
+  end
+end
+
+task :start_qp do
+  on roles(:all) do
+    execute "cd #{fetch(:deploy_to)}/current/script && chmod a+x * && source start_qp.sh"
```

Comment:
Because the script is started from within the current/script directory, the `bin/bundle` commands in the script don't work.  Recommend changing to the current/ directory and running script/start_qp.sh